app/controllers/api/database.php

```diff
@@ -224,79 +320,289 @@
         $response->noContent();
     });
 
-App::post('/v1/database/collections/:collectionId/attributes')
-    ->desc('Create Attribute')
+App::post('/v1/database/collections/:collectionId/attributes/string')
+    ->desc('Create String Attribute')
     ->groups(['api', 'database'])
     ->label('event', 'database.attributes.create')
     ->label('scope', 'attributes.write')
     ->label('sdk.namespace', 'database')
     ->label('sdk.platform', [APP_PLATFORM_SERVER])
```

Comment:
We have stopped using the `sdk.platform` label on Appwrite 0.8. Please use `sdk.auth` instead, some relevant examples can be found in the users service controller.

Reply:
Addressed in 3846d66